CadRevealFbxProvider.Tests/FbxNodeToCadRevealNodeConverterTests.cs

```diff
@@ -0,0 +1,60 @@
+namespace CadRevealFbxProvider.Tests;
+
+using CadRevealComposer;
+using CadRevealComposer.Configuration;
+using CadRevealComposer.IdProviders;
+using CadRevealComposer.Operations;
+using CadRevealComposer.Primitives;
+using System.Numerics;
+
+[TestFixture]
+public class FbxNodeToCadRevealNodeConverterTests
+{
+    private static readonly string TestFile = new("TestSamples\\cube_and_instanced_cube_with_parent.fbx");
+
+    [Test]
+    public void CubeAndInstancedCubeParentedToBaseMeshAllWithTransforms_ConvertRecursive_VerifyCorrectTransformations()
```

Comment:
Consider adding a few comments on the expectations with inline comments in this test. I found it a bit hard to understand while reading.